onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selector_action_transformer.cc

```diff
@@ -45,6 +45,29 @@ void DropQDQNodesRules(SelectorActionRegistry& qdq_selector_action_registry) {
 #endif
 }
 
+// create rules for ops that don't change the data
+void DropDQNodesRules(SelectorActionRegistry& qdq_selector_action_registry) {
+  // 2 nodes. DQ, target. Merge into target and remove DQ.
+  const std::string action_name{"dropDQ"};
+  NTO::NodeLocation dq{NTO::NodeType::kInput, 0};
+
+  // Move DQ input 0 to target input 0.
+  std::vector<NodeAndMoveInfo> moves{
+      MoveToSlot(dq, ArgType::kInput, 0, ArgType::kInput, 0)};
+
+  std::unique_ptr<Action> action = std::make_unique<MergeIntoTarget>(std::move(moves));
+
+#if !defined(ORT_MINIMAL_BUILD)
+  std::unique_ptr<NodeSelector> selector = std::make_unique<QDQ::DropDQNodesSelector>();
+  qdq_selector_action_registry.RegisterSelectorAndAction(action_name,
+                                                         {{"ArgMax", {}}},
```

Comment:
I just found that int8 and uint8 for ArgMax were only added for opset13 in your last PR. you also need to register for opset 1,11,12

Reply:
It would be in another follow-up PR